be/src/exec/pipeline/pipeline_builder.h

```diff
@@ -55,13 +58,22 @@ class PipelineBuilderContext {
 
     FragmentContext* fragment_context() { return _fragment_context; }
 
+    size_t get_dop_of_scan_node(TPlanNodeId id) const {
+        if (_per_scan_node_dop.count(id)) {
+            return _per_scan_node_dop.at(id);
```

Comment:
```suggestion
            return _per_scan_node_dop[id];
```

Reply:
get_dop_of_scan_node is a const-modified function, operator[] is not const, instead , at(size_t i) is its const counterpart.